### PR TITLE
PostmanTests_CreateUser

### PR DESCRIPTION
**Pull request by ifelseelif**

### CODE REVIEW

**Review by Cerberous007 — APPROVED** [review]

File: PostmanTests/UserService.postman_collection.json [hunk 1]
```diff
@@ -50,6 +50,144 @@
...
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n\t\"Email\" : \"email@yandex.ru\",\n\t\"FirstName\" : \"Leo\",\n\t\"LastName\" : \"Kor\",\n\t\"Birthday\" : \"2020-04-05\"\n}",
```
Comment (strannick-rodion) [inline PostmanTests/UserService.postman_collection.json:126]: Why didn't you write the password field empty, and in the next test you wrote everything

**Review by strannick-rodion — APPROVED** [review]

File: PostmanTests/UserService.postman_collection.json [hunk 2]
```diff
@@ -50,6 +50,144 @@
...
+							"mode": "raw",
+							"raw": "{\n\t\"Email\" : \"\",\n\t\"Password\" : \"\",\n\t\"FirstName\" : \"\",\n\t\"LastName\" : \"\",\n\t\"Birthday\" : \"\"\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "https://localhost:5011/users/create",
```
Comment (dTarasova) [inline PostmanTests/UserService.postman_collection.json]: it is very soon (if no already) will be user/create

File: PostmanTests/UserService.postman_collection.json [hunk 3]
```diff
@@ -50,6 +50,144 @@
...
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n\t\"Email\" : \"email2@yandex.ru\",\n\t\"Password\" : \"123\",\n\t\"FirstName\" : \"Lep\",\n\t\"LastName\" : \"Kor\",\n\t\"Birthday\" : \"2020-04-05\"\n}",
```
Comment (dTarasova) [inline PostmanTests/UserService.postman_collection.json]: difficult to read

File: PostmanTests/UserService.postman_collection.json [hunk 4]
```diff
@@ -50,6 +50,144 @@
...
+							"path": [
+								"users",
+								"create"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Body with empty fields",
```
Comment (dTarasova) [inline PostmanTests/UserService.postman_collection.json:144]: If only one field will be missed who have to check this? Unit or postman tests?

File: PostmanTests/UserService.postman_collection.json (hunk 3 above)
Comment (DarkFelas) [reply PostmanTests/UserService.postman_collection.json]: it is how postman format collection, no need for any changes here

File: PostmanTests/UserService.postman_collection.json (hunk 2 above)
Comment (nmamro) [reply PostmanTests/UserService.postman_collection.json]: > it is very soon (if no already) will be user/create

Nope, it won't xD

File: PostmanTests/UserService.postman_collection.json (hunk 1 above)
Comment (ifelseelif) [reply PostmanTests/UserService.postman_collection.json:126]: One test checks to see if a field is skipped. Other if all fields are empty.

File: PostmanTests/UserService.postman_collection.json (hunk 4 above)
Comment (ifelseelif) [reply PostmanTests/UserService.postman_collection.json:144]: An extra test won't hurt. This test is to verify that the validator is working.However, it is not merged yet

**Review by nmamro — APPROVED** [review]
Seems ok to me

**Review by dTarasova — APPROVED** [review]

**Pull request merged**